grammars/html.cson

```diff
@@ -363,7 +363,7 @@
   'entities':
     'patterns': [
       {
-        'begin': '(&)([a-zA-Z0-9]+|#[0-9]+|#x[0-9a-fA-F]+)'
+        'begin': '(&)([a-zA-Z0-9]+|#[0-9]+|#x[0-9a-fA-F]+)(?=;)'
```

Comment:
I think that's the better way forward since entities aren't able to wrap around anything.